### ISSUE TITLE
Summary scalar values are not appearing on Tensorboard

### ISSUE THREAD

**Issue by kasrahbar** [tensorflow/tensor2tensor#984]
I've noticed that certain summary scalar values that used to appear are no longer appearing on Tensorboard on my language modeling problem. Specifically, "input stats" which include batch size and padding fraction are no longer showing up. This issue persists across several different models including transformer, transformer_moe and universal transformer. This also includes model-specific scalar values like the "ponder time" for universal transformer. 

On the other hand, scalar values like loss, learning rate, and global step/sec are showing up just fine. Other summary types like image summaries and histograms show up as expected.

Any ideas what this might be due to? Perhaps it's an issue on the Tensorflow side (tf.contrib.learn)?
...

### Environment information

```
OS: Ubuntu 18.04.1 LTS

$ pip freeze | grep tensor
tensor2tensor==1.7.0
tensorboard==1.9.0
tensorflow-gpu==1.9.0


$ python -V
Python 2.7.12

**Comment by MostafaDehghani** [comment]
@kasrahbar, wonder if you can share what was the problem and the solution if you already solved it (since you closed the issue)?

**Comment by kasrahbar** [comment]
Sure. I ended up switching to using tf.summary.histogram instead of scalars, for example to visualize ponder times for universal transformer, which I find more useful anyway. I'm still not sure why scalars won't work but histograms do work, but maybe it's something specific to my setup.

Nice work on the universal transformer btw.

**Comment by MostafaDehghani** [comment]
Thanks @kasrahbar!
I see....asked because someone else had the same issue with seeing model specific scalar summaries on tensorboard and I had no idea what might go wrong to cause this. 

Thanks :)
